content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -139,4 +161,21 @@ For example: to enable deployments and daemonsets, set
 
 {{< note >}}Individual resource enablement/disablement is only supported in the `extensions/v1beta1` API group for legacy reasons.{{< /note >}}
 
+## Persistence
+
+Kubernetes stores its serialized state in terms of the API resources; the persistence store
+is {{< glossary_tooltip term_id="etcd" >}} although it's possible to change the code to
```

Comment:
nit: Leave out the phrase, "change the code to implement ...",
The default persistence store is etcd.

Reply:
Yes and we don't recommend "changing the code" like that. If anything, one can use the adapter k3s built.

Reply:
I wanted to highlight that changing away from _etcd_ is feasible but not easy.

Reply:
I would say it is definitely not feasible like that, especially not for end users. If people don't want to run `etcd`, they should look into a shim, like https://github.com/rancher/kine

Reply:
I've simplified, think that addresses concerns.